components/controls/Toggle.tsx

```diff
@@ -36,7 +37,7 @@ export default class Toggle extends React.PureComponent<Props> {
   };
 
   handleClick = () => {
-    if (this.props.onChange) {
+    if (this.props.onChange && !this.props.disabled) {
```

Comment:
@philippe-perrin-sonarsource You should not need to do this.

Reply:
Try `<button disabled onClick={() => alert('plop')}>click me!</button>`, onClick is not called.

Reply:
You're damn right. Good catch.